src/main/java/com/example/bookstore/validation/Email.java

```diff
@@ -0,0 +1,20 @@
+package com.example.bookstore.validation;
+
+import com.example.bookstore.exception.RegistrationException;
+import com.example.bookstore.validation.validators.EmailValidator;
+import jakarta.validation.Constraint;
+import jakarta.validation.Payload;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Constraint(validatedBy = EmailValidator.class)
+@Target({ElementType.PARAMETER, ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Email {
```

Comment:
There is a built-in `Email` annotation for validation. You can leave this custom annotation here, but so you know in future.